src/js/rx/containers/Active.jsx

```diff
@@ -22,6 +24,15 @@ class Active extends React.Component {
   constructor(props) {
     super(props);
     this.handleSort = this.handleSort.bind(this);
+
+    this.checkWindowSize = _.debounce(() => {
+      if (document.documentElement.clientWidth < 600) {
```

Comment:
I just thought of another way to do this. We could assign an ID to the view toggle element and then instead of checking window width, check if that element is visible in the DOM. The view toggle element should be hidden automatically by CSS classes (`display:none`)